### ISSUE TITLE
Failed to Initialize WebNnRoom -- ETIMEDOUT... 

### ISSUE THREAD

**Issue by zipzit** [ibelem/webnnmeeting#1]
Failed to intialize webnnroom Error: connect ETIMEDOUT 10.239.47.52:3000
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1126:14) {
  errno: 'ETIMEDOUT',
  code: 'ETIMEDOUT',
  syscall: 'connect',
  address: '10.239.47.52',
  port: 3000
}

Windows 10 home OS, version 1909
Intel i7-3770, 12.0GB ram,  64 bit o/s
$ npm --version  // 6.10.3
$ node --version // v12.10.0
I'm using github bash CLI client terminal...

I will say when I first ran `npm install` there were a whole lot of vulnerability warnings.  `npm update` and `npm audit fix` corrected most of them, rerun `npm run dev` = same error as above.  

Am i doing this correctly?  many thanks...

**Comment by ibelem** [comment]
@zipzit Sorry I am late, I just added [Server.md](https://github.com/ibelem/webnnmeeting/blob/master/doc/Server.md), please follow this page to setup the WebRTC server at first then fill the WebRTC server IP, id and key in this configuration.

**Comment by lordmen99** [comment]
any demo live from this app?

**Comment by ibelem** [comment]
Thanks @lordmen99 for the interests.  The demo requires a WebRTC server and I have no plans to pay for this service from Amazon AWS or Google Cloud but it can be setup easily (please refer to [Server.md](https://github.com/ibelem/webnnmeeting/blob/master/doc/Server.md)) on local PC. 

This app is part of W3C [WebNN API examples](https://github.com/intel/webml-polyfill/tree/master/examples) now, you can try the [live Semantic Segmentation example](https://intel.github.io/webml-polyfill/examples/semantic_segmentation/?prefer=none&b=WebGL&m=deeplab_mobilenet_v2_224_atrous_tflite&s=image&d=0&f=WebNN) for machine learning features.

WebNN Meeting will look like:

<img src="https://github.com/ibelem/webnnmeeting/raw/master/doc/screenshot1.png" alt="WebNN Meeting" width="60%">
<img src="https://github.com/ibelem/webnnmeeting/raw/master/doc/screenshot2.png" alt="WebNN Meeting" width="60%">

**Comment by lordmen99** [comment]
its have features record when we in live?

**Comment by ibelem** [comment]
@lordmen99 No, I didn't add it, but it shouldn't be hard to have the video recording feature. We can use Media Recorder API and WebRTC for video recording.

https://blog.addpipe.com/html5-video-recording-webrtc-mediarecorder-api/ 
https://addpipe.com/media-recorder-api-demo/

**Comment by lordmen99** [comment]
ok, then for background that can use image,video or office file? like presentation

**Comment by ibelem** [comment]
@lordmen99 Currently it can use images. Background video is a good suggestion and shoule be cool but I have concerns about the performance. Users need to select video clips then apply as the conference background, I haven't seen Video Conference products like Zoom and MS Teams are using video clips for background replacement.
